src/solvers/prop/cover_goals.cpp

```diff
@@ -11,11 +11,12 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "cover_goals.h"
 
-#include <util/message.h>
-
 #include "literal_expr.h"
 #include "prop_conv.h"
 
+#include <util/message.h>
```

Comment:
I believe most of the files now have directory-local includes last, so I'd rather keep this include where it was?

Reply:
Sure, moved